ads/yandex.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Yandex
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="yandex"
+    data-block-id="R-A-123456-1">
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
+
+
+Required parameters:
+- data-block-id
+
+Supported parameters:
+- data
+- isAdfox
```

Comment:
hmmm `data-isAdfox` will be converted to `data.isadfox`  Do you mean `data-is-adfox` here?